pkg/rpc/response/result/block.go

```diff
@@ -15,19 +17,9 @@ type (
 	// Tx wrapper used for the representation of
 	// transaction on the RPC Server.
 	Tx struct {
-		TxID       util.Uint256            `json:"txid"`
-		Size       int                     `json:"size"`
-		Type       transaction.TXType      `json:"type"`
-		Version    uint8                   `json:"version"`
-		Attributes []transaction.Attribute `json:"attributes"`
-		VIn        []transaction.Input     `json:"vin"`
-		VOut       []transaction.Output    `json:"vout"`
-		Scripts    []transaction.Witness   `json:"scripts"`
-
-		SysFee util.Fixed8 `json:"sys_fee"`
-		NetFee util.Fixed8 `json:"net_fee"`
-
-		Nonce uint32 `json:"nonce,omitempty"`
+		Transaction *transaction.Transaction
```

Comment:
Not sure it's a good change, we're missing original `TxID` and `Size` field, although it's possible to get them via other means. @im-kulikov: are you using any of these fields directly?